FF1Randomizer/MainWindow.xaml.cs

```diff
@@ -2,6 +2,7 @@
 using Microsoft.Win32;
 using RomUtilities;
 using System;
+using System.Diagnostics;

```

Comment:
These changes are mostly meaningless.

Reply:
was required for Stopwatch, i will remove theese, when a merge draws nearer.